moabb/datasets/download.py

```diff
@@ -48,12 +48,9 @@ def data_path(url, sign, path=None, force_update=False, update_path=True, verbos
         of length one, for compatibility.
 
     """  # noqa: E501
-    sign = sign.upper()
-    key = "MNE_DATASETS_{:s}_PATH".format(sign)
+    path = get_dataset_path(sign, path)
+    key = "MNE_DATASETS_{:s}_PATH".format(sign.upper())
     key_dest = "MNE-{:s}-data".format(sign.lower())
-    if get_config(key) is None:
-        set_config(key, osp.join(osp.expanduser("~"), "mne_data"))
-    path = _get_path(path, key, sign)
     destination = _url_to_local_path(url, osp.join(path, key_dest))
     # Fetch the file
     if not osp.isfile(destination) or force_update:
```

Comment:
You could remove the _do_path_update(path, update_path, key, sign) on line 64 (and the comment before), it is no longer needed. So key variable is not needed outside `get_dataset_path`you could remove it.